src/ucp/tag/rndv.c

```diff
@@ -94,10 +102,13 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_rtr, (self),
                  uct_pending_req_t *self)
 {
     ucp_request_t *rndv_req = ucs_container_of(self, ucp_request_t, send.uct);
+    size_t packed_rkey_size;
     ucs_status_t status;
 
     /* send the RTR. the pack_cb will pack all the necessary fields in the RTR */
-    status = ucp_do_am_bcopy_single(self, UCP_AM_ID_RNDV_RTR, ucp_tag_rndv_rtr_pack);
+    packed_rkey_size = ucp_ep_config(rndv_req->send.ep)->tag.rndv.rkey_size;
```

Comment:
why just not set it directly, i.e.
```
status = ucp_do_am_single(self, UCP_AM_ID_RNDV_RTR, ucp_tag_rndv_rtr_pack,
                          sizeof(ucp_rndv_rtr_hdr_t) +
                          ucp_ep_config(rndv_req->send.ep)->tag.rndv.rkey_size);
```
?

cosmetic change?

Reply:
yes, to fit into 80 chars